components/brave_wallet/browser/blockchain_list_parser.cc

```diff
@@ -99,6 +99,8 @@ bool ParseTokenList(const std::string& json,
     else
       blockchain_token->is_erc721 = false;
 
+    blockchain_token->is_nft = blockchain_token->is_erc721;
```

Comment:
We don't have is_nft used by our token maps from wallet component extension atm.